package.json

```
@@ -30,8 +30,8 @@
     "posttest": "aud --production",
     "test:ci": "npm run jest -- --ci --runInBand",
     "jest": "jest --coverage __tests__/**/*",
-    "generate-list-of-rules": "md-magic --path '**/*.md' --ignore 'node_modules'",
-    "generate-list-of-rules:check": "npm run generate-list-of-rules && git diff --exit-code README.md"
+    "generate-list-of-rules": "eslint-doc-generator --rule-doc-title-format name",
```

if this needs the build script, then:
```suggestion
    "pregenerate-list-of-rules": "npm run generate-list-of-rules",
    "generate-list-of-rules": "eslint-doc-generator --rule-doc-title-format name",
```

---

I think you meant:

```
"pregenerate-list-of-rules": "npm run build",
```

Updated.

---

right yes sorry :-)